### DISCUSSION TITLE
UI\Add new card

### DISCUSSION THREAD

**Discussion by KaherdinTristan** [fuatakgun/eufy_security#79] [Q&A] [answered]
Hi
Could someone make a post as to how to create the example in UI-mode (now YAML code in the ui-lovelace.yaml-file)

Im thinking of this example
https://github.com/fuatakgun/eufy_security#63-optional---webrtc

**Comment by dpomnean** [comment]
You can view some examples and the process in this closed issue

#43

**Reply by fuatakgun** [reply]
Also, latest section of readme was updated accordingly

**Comment by KaherdinTristan** [comment, marked as answer]
The specific question I have is for a code example to use in the UI, not in the YAML, see example below. 
I Could not find any such example. The Dashboard I use to test this in is not in YAML-mode. 
![image](https://user-images.githubusercontent.com/3398594/137362874-ae1844af-775e-4c92-ba74-755d78174278.png)

**Reply by fuatakgun** [reply]
it is not clear to me what is your ask.

I have created a new view and added a conditional card and clicked on `show code editor` to update the code.

**Comment by janneHalttu** [comment]
Did you get this to work? I only get 150x100 size card with example code provided by quide.. small.. 
So is there now way to get high res stream?
